app/Providers/AppServiceProvider.php

```diff
@@ -30,6 +31,7 @@ public function register()
     {
         $this->app->singleton(Environment::class);
         $this->app->singleton(Process::class);
+        $this->app->singleton(CommandExecutor::class);
```

Comment:
i had to add this so we keep `disable state` on `articalCallNoOutput`

Reply:
even though having artisanCallNoOutput anymore, I guess it is nice to keep it as singleton with this setting.